runtime/westend/src/weights/xcm/mod.rs

```diff
@@ -0,0 +1,191 @@
+mod pallet_xcm_benchmarks_fungible;
+mod pallet_xcm_benchmarks_generic;
+
+use crate::Runtime;
+use frame_support::weights::Weight;
+use sp_std::prelude::*;
+use xcm::{latest::prelude::*, DoubleEncoded};
+
+use pallet_xcm_benchmarks_fungible::WeightInfo as XcmBalancesWeight;
+use pallet_xcm_benchmarks_generic::WeightInfo as XcmGeneric;
+
+pub enum AssetTypes {
+	Balances,
+	Unknown,
+}
+
+impl From<&MultiAsset> for AssetTypes {
+	fn from(asset: &MultiAsset) -> Self {
+		match asset {
+			MultiAsset { id: Concrete(MultiLocation { parents: 0, interior: Here }), .. } =>
+				AssetTypes::Balances,
+			_ => AssetTypes::Unknown,
```

Comment:
shouldn't be `unreachable!`?

Reply:
No, this is where someone sends us an XCM with an asset that we are not familiar with. We will make the XCM invalid, but we def do not want to panic.